go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	github.com/moby/sys/signal v0.5.1-0.20210723232958-8a51b5cc8879
 	github.com/moby/sys/symlink v0.1.0
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.0.1
+	github.com/opencontainers/image-spec v1.0.2-0.20210819154149-5ad6f50d6283
```

Comment:
Do we have an issue ticket to discuss releasing v1.0.2?
(I thought we had one, but can't find)

Reply:
I expect we can import the tagged version when https://github.com/opencontainers/image-spec/issues/803 is resolved.